sklearn/metrics/cluster/supervised.py

```diff
@@ -65,6 +65,8 @@ def contingency_matrix(labels_true, labels_pred, eps=None, max_n_classes=5000):
         This help to avoid Memory error with regression target
         for mutual_information.
 
+        .. versionadded:: 0.18
```

Comment:
I don't think this is true. It was maybe added to the docs recently but it has been here for a while.